packages/react/react/src/hooks/useOutsideClickEffect.spec.tsx

```diff
@@ -42,13 +42,14 @@ describe('useOutsideClickEffect', () => {
   }
 
   it('컨테이너 바깥에 위치한 DOM에서 이벤트가 발생하면 콜백이 호출된다.', async () => {
+    const user = userEvent.setup();
     const onEffect = jest.fn();
     prepare({ onEffect });
 
-    userEvent.click(screen.getByTestId('container-0'));
```

Comment:
Starting with v14, it is recommended to add "await" to multiple guides.

### reference
https://github.com/testing-library/user-event/discussions/1050
https://testing-library.com/docs/user-event/convenience